nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/test/java/org/apache/nifi/provenance/ITestPersistentProvenanceRepository.java

```diff
@@ -1250,17 +1135,11 @@ public void authorize(ProvenanceEventRecord event, NiFiUser user) {
 
         repo.waitForRollover();
 
-        try {
-            repo.getEvent(0L, null);
-            Assert.fail("getEvent() did not throw an Exception");
-        } catch (final Exception e) {
-            Assert.assertSame(expectedException, e);
-        }
+        assertThrows(expectedException.getClass(), () -> repo.getEvent(0, null));
     }
 
     @Test
     public void testNotAuthorizedGetEventRange() throws IOException {
-        assumeFalse(isWindowsEnvironment());
```

Comment:
Why no DisabledOnOs?

Reply:
As mentioned in the other note, I could add the annotation at the class level, but it seems unnecessary now that this is an integration test instead of a unit test.